controllers/index.js

```diff
@@ -140,7 +145,9 @@ exports.index = (req, res) => {
         }
     }).catch((err) => {
         console.log(err);
-        if (req.xhr) {
+        if (err === 'notUser') {
+            res.status(400).render('partials/error-user', {layout: false});
```

Comment:
What happens if you hit refresh having put in an organization name? Does it display a white page with just the error in black? If so, I think this needs to be within the `if (req.xhr)` conditional.